Sources/Engine/Graphics/Texture.cpp

```diff
@@ -1614,12 +1614,6 @@ void ProcessScript_t( const CTFileName &inFileName) // throw char *
   INDEX i=0;
   FOREACHINLIST( CFileNameNode, cfnn_Node, FrameNamesList, it1)
   {
-    if( i != 0) {   // we have to skip first picture since it has already been done
-      inPic.LoadAnyGfxFormat_t( CTString(it1->cfnn_FileName));
-      // add picture as next frame in texture
-      tex.AddFrame_t( &inPic);
-      inPic.Clear();
-    }
     i++;
```

Comment:
i is incremented here, so it will be different of 0 after first loop. The FOREACHINLIST macro is a `for` loop.

Reply:
Right. Sorry. I have reverted this change.